src/lib/dev_types.ts

```diff
@@ -0,0 +1,9 @@
+export type Jubmoji = {
```

Comment:
Can we have comments explaining that R/T/U are serialized, that the sig is DER format, etc.